src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -1621,7 +1621,7 @@ def _get_pp_attr(self, table: str, attribute: str, default: Optional[Union[float
         attr_data = pp_component_data[attribute]
 
         # If any of the attribute values are missing, and a default is supplied, fill the nans with the default value
-        nan_values = np.equal(attr_data, None) if attr_data.dtype is np.dtype("O") else np.isnan(attr_data)
+        nan_values = np.equal(attr_data, None) if attr_data.dtype is np.dtype("O") else np.isnan(attr_data)  # type: ignore
```

Comment:
Do you know which type issue arises? Is there another way to solve it?

Reply:
To solve this, I have written 5 options in the original ticket.
For now, I think I implemented the simplest one, to let mypy ignore this single error.

Reply:
@bramstoeller, this (sonar project properties) slips in the repo with automatic merges (instead of cherry picks).

One way to solve this issue is to ensure that you have the correct version in git, and then add it to the .gitignore, so updates won't get committed/pushed. (AFAIK, git will keep the original version).